File: Data Structures & Algorithms/max-consecutive-ones/submission-1.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        max_cur, cur = 0, 0
        for v in nums:
            if v == 1:
                cur += 1
            else:
                max_cur = max(cur, max_cur)
                cur = 0

        return max(cur, max_cur)

File: Data Structures & Algorithms/remove-element/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} val
     * @return {number}
     */
    removeElement(nums, val) {
        let l = 0
        let r = nums.length

        for (;l < r;) {
            if (nums[l] == val) {
                r--
                nums[l] = nums[r]
            } else {
                l++
            }
        }

        return l

    }
}
